app/src/main/java/com/example/cmput301f18t26/icare/Controllers/DataController.java

```diff
@@ -1,12 +1,15 @@
 package com.example.cmput301f18t26.icare.Controllers;
 
 class DataController {
-    private static final DataController ourInstance = new DataController();
 
-    static DataController getInstance() {
-        return ourInstance;
+    public DataController() {
```

Comment:
If you choose to mock out the private DS's, this constructor could be mocked with various types of inputs. Depending how our DataController will be used, you can use method overloading here to initialize the controller with different types of data depending on whats available at the time we construct it.